core/include/core/Experiments.h

```diff
@@ -0,0 +1,11 @@
+#ifndef _EXPERIMENTS_H
+#define _EXPERIMENTS_H
+
+namespace Experiments {
+        const unsigned SPT = 0;
+        const unsigned PB = 1;
+        const unsigned BK = 2;
+}
```

Comment:
If it is only used in one place, you could probably make do with this being a string as well. It at least does not need to be in a header file.